receiver/prometheusremotewritereceiver/receiver_bench_test.go

```diff
@@ -0,0 +1,39 @@
+package prometheusremotewritereceiver
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+// makeLabels builds a labels.Labels with the given total number of labels.
+// It always includes "job", "instance", and the metric name label, plus (total-3) extra labels.
+func makeLabels(total int) labels.Labels {
+	if total < 3 {
+		total = 3
+	}
```

Comment:
Why total-3? it's because we set 3 on `extractAttributes`, if it was, can we add a comment explaining the why's?